### ISSUE TITLE
search function doesn't work

### ISSUE THREAD

**Issue by voncloft** [venomlinux/scratchpkg#10]
when I type scratch search kate it says function is not available

**Comment by emmett1** [comment]
Hey, i’m sorry about that. I’m separated some of ‘scratch’ function to ‘pkgquery’.
Run ‘scratch help’ and ‘pkgquery help’ to see options.

**Comment by voncloft** [comment]
I tried pkgquery search it works but the color coding does not work its just white text

**Comment by emmett1** [comment]
Yes, i did not make ‘pkgquery’ colorize, only ‘scratch’ has colorize.

**Comment by voncloft** [comment]
Curious as to why pkguery would not have color...just makes more sense to categorize the text.

**Comment by emmett1** [comment]
Bcoz I wrote ‘pkgquery’ in more simpler format. And ‘scratch’ is a little mess up get me confuse sometimes to maintain it. Thats why i’m sparating most of ‘quering’ function. If this affect users, i might bring back ‘search’ function to ‘scratch’.

**Comment by voncloft** [comment]
Each to their own.... luckily I had a backup of scratch that still uses search and overwrote the version on this pc

**Comment by emmett1** [comment]
Old scratch is very different from new one. Give me like few hours, i’ll try bring back search function to scratch.

**Comment by emmett1** [comment]
Hey, i'm bring back old search function to scratch, maybe a little bit different bcoz its posix now, but the output is same.
https://github.com/venomlinux/scratchpkg/commit/3b7a73c0d813c18cd21e13ab2e2d8876c1b6d62d

**Comment by voncloft** [comment]
It works thanks again.

**Comment by emmett1** [comment]
No problem. :D
